vignettes/dependencies.Rmd

```diff
@@ -52,6 +52,11 @@ A type of 'github' can be specified, but is not required
 Remotes: github::hadley/ggplot2
 ```
 
+Remember that the `Remotes:` field only specifies the *source* of a package, but it does not actually make it available to the R session.
+You still also have to call the package in `Imports:`, `Suggests:` `Depends:` or  `LinkingTo:`.
+In the above example, where the *package name* and the *repo name* are identical, you still need `Imports: ggplot2` in your `DESCRIPTION`.
+
```

Comment:
I think something like the following would be clearer. The distinction between package name and repo name while correct is fairly rare and just muddies the waters.

`Remotes:` specifies the *source* of a development package, the package still needs to be listed in `Imports:`, `Suggests:` `Depends:` or  `LinkingTo:`. e.g. `Imports: ggplot2` in the above example.